sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -185,21 +185,21 @@ internal class AmqpReceiver : TransportReceiver
                         new DateTime(lockedUntilUtcTicks, DateTimeKind.Utc)
                         : DateTime.MinValue;
 
-                        var source = (Source)link.Settings.Source;
-                        if (!source.FilterSet.TryGetValue<string>(AmqpClientConstants.SessionFilterName, out var tempSessionId))
-                        {
-                            link.Session.SafeClose();
-                            throw new ServiceBusException(true, Resources.SessionFilterMissing);
-                        }
```

Comment:
When I saved the file, it fixed the indentation. It was incorrectly 1 level too deep.